src/common/connect.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { ConnectedHost } from './ConnectedHost';
+
+export function connect<T>(
+  component: React.ReactType,
+  storesToSubscribe: string[],
+  getProps: (props: T, ...stores) => T
```

Comment:
We should enforce that you pass in `StoreKey<B>` instances and that `stores` is an `IStoreCollection`.